docs/index.html

```diff
@@ -39,7 +34,7 @@
         </div>
       </nav>
 
-      <section class="jumbotron jumbotron-fluid d-flex align-items-center">
+      <section class="bg-body-tertiary py-4 d-flex align-items-center">
         <div class="container text-center">
           <div class="row justify-content-center">
             <div class="col-11 col-sm-10 col-md-8 animated fadeInUp">
```

Comment:
Looks like you're using the animate.css library. You probably want to add the 
Looks like you're using the animate css library, but it's not working.

```suggestion
            <div class="col-11 col-sm-10 col-md-8 animated fadeInUp">
```

Reply:
Looks like some of your changes took care of this.